src/backend/InvenTree/InvenTree/models.py

```diff
@@ -526,41 +526,36 @@ def create_attachment(self, attachment=None, link=None, comment='', **kwargs):
 
 
 class InvenTreeTree(MetadataMixin, PluginValidationMixin, MPTTModel):
```

Comment:
Q: Is there a technical reason to have the Validation Mixin here? If not, I would recommend removing it to reduce unintended nesting

Reply:
No, there's no reason. I have now flattened it out